tools/submission/power/power_checker.py

```diff
@@ -408,6 +408,7 @@ def get_avg_power(power_path: str, run_path: str) -> Tuple[float, float]:
 
         with open(spl_fname) as f:
             for line in f:
+                if not line.startswith("Time"): continue
```

Comment:
Hi @psyhtest since this file is automatically synced from the power-dev repository, can you please give this update in power-dev repository?

Reply:
This PR updated. A companion power-dev PR created https://github.com/mlcommons/power-dev/pull/325